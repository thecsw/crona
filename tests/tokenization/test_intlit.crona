int i = 10
int j = -10000
int k = 9999999999
int l = -9999999999
-
-145
235