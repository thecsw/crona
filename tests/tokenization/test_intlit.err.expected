FATAL [3,9]: Integer literal too large;  using max value
FATAL [4,10]: Integer literal too large;  using max value
